src/itype.h

```diff
@@ -42,9 +42,9 @@ std::string const& default_ammo(std::string const &guntype);
 struct explosion_data {
     // Those 4 values are forwarded to game::explosion.
     int power    = -1;
```

Comment:
This should probably be a `float`, as it is forwarded to `game::explosion`, which expects a `float`.